src/templates/_macros/market_button.html

```diff
@@ -27,7 +27,7 @@
           {{ _('Installed') }}
         {% elif installed %}
           {{ _('Open') }}
-        {% elif user.has_installed(app.id) or user.has_purchased(app.id) %}
+        {% elif user.has_installed(app.id) or user.has_purchased(app.id) or app.role == 'langpack' %}
```

Comment:
@ngokevin : since we check for `installed` above (which uses the manifest url instead of the id to make it compatible with langpacks), do we need the first part of the `elif` clause here ? I don't think it does anything, unless I missed something obvious.